client.go

```diff
@@ -75,7 +81,13 @@ func RawRequest(URL string, r io.Reader) (*http.Response, error) {
 		return nil, errors.New("Refusing to send OFX request with possible plain-text password over non-https protocol")
 	}
 
-	response, err := http.Post(URL, "application/x-ofx", r)
+	var response *http.Response
+	var err error
+	if strings.HasPrefix(URL, "https://ofx.discovercard.com") {
```

Comment:
I dislike the hard-coded detection here, but fixing this will require more major structural changes